examples/speak/Speak.csproj

```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
-		<TargetFramework>net8.0</TargetFramework>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
same comment as above

Reply:
Please see previous comment. we should be good!